script/shoryuken

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec bin/shoryuken --rails -r app/jobs/sqs_worker.rb
```

Comment:
@qrush:
> This should go in bin/ now

Since `bin` is gitignored, it seemed like a non-starter executables like this. Any suggestions?